src/components/toolbox/box/infoText.css

```diff
@@ -0,0 +1,15 @@
+@import '../../app/variables.css';
+@import '../../app/mixins.css';
+
+.infoText {
+  @mixin contentLargest;
+
+  letter-spacing: 0.21px;
+  line-height: 25px;
```

Comment:
do you need to set these in here, I think that the purpose of the mixin is for not to do it.
I am talking about letter-spacing and line-height